recipes/platform.delegates/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class PlatformInterfacesConan(ConanFile):
+    name = "platform.delegates"
+    license = "MIT"
+    homepage = "https://github.com/linksplatform/Delegates"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = """platform.delegates is one of the libraries of the LinksPlatform modular framework, which uses 
+    innovations from the C++17 standard, for easier use delegates/events in csharp style."""
+    topics = ("linksplatform", "cpp17", "delegates", "events", "header-only")
+    settings = "os", "compiler", "build_type", "arch"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _subfolder_sources(self):
+        return os.path.join(self._source_subfolder, "cpp", "Platform.Delegates")
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "8",
+            "Visual Studio": "16",
+            "clang": "10",
+            "apple-clang": "10"
+        }
+
+    def validate(self):
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if tools.Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration("platform.delegates/{} "
+                                            "requires C++17 with {}, "
+                                            "which is not supported "
+                                            "by {} {}.".format(self.version, self.settings.compiler, self.settings.compiler, self.settings.compiler.version))
```

Comment:
We typically print a warning for the consumer when the compiler is unknown, you can see how with conan-io/conan#8002

Reply:
Secret unknown compiler with c++17 standard support. Okay

Reply:
It's because consumers can have custom settings and use whatever name they like for a compiler. we should not block their workflows